src/host/_stream.cpp

```diff
@@ -330,59 +330,12 @@ constexpr unsigned int LOCAL_BUFFER_SIZE = 100;
             // Only act on a delayed EOL if we didn't move the cursor to a different position from where the EOL was marked.
             if (coordDelayedAt.X == CursorPosition.X && coordDelayedAt.Y == CursorPosition.Y)
             {
-                bool fDoEolWrap = false;
-
-                if (WI_IsFlagSet(dwFlags, WC_DELAY_EOL_WRAP))
-                {
-                    // Correct if it's a printable character and whoever called us still understands/wants delayed EOL wrap.
-                    if (*lpString >= UNICODE_SPACE)
-                    {
-                        fDoEolWrap = true;
-                    }
-                    else if (*lpString == UNICODE_BACKSPACE)
-                    {
-                        // if we have an active wrap and a backspace comes in, process it by moving the cursor
-                        // back one cell position unless it's already at the start of a row.
-                        *pcb += sizeof(WCHAR);
-                        lpString++;
-                        pwchRealUnicode++;
-                        if (CursorPosition.X != 0)
-                        {
-                            --CursorPosition.X;
-                            Status = AdjustCursorPosition(screenInfo, CursorPosition, WI_IsFlagSet(dwFlags, WC_KEEP_CURSOR_VISIBLE), psScrollY);
-                            CursorPosition = cursor.GetPosition();
-                        }
-                        continue;
-                    }
-                }
-                else
-                {
-                    // Uh oh, we've hit a consumer that doesn't know about delayed end of lines. To rectify this, just quickly jump
-                    // forward to the next line as if we had done it earlier, then let everything else play out normally.
-                    fDoEolWrap = true;
-                }
-
-                if (fDoEolWrap)
-                {
-                    CursorPosition.X = 0;
-                    CursorPosition.Y++;
+                CursorPosition.X = 0;
+                CursorPosition.Y++;
 
-                    Status = AdjustCursorPosition(screenInfo, CursorPosition, WI_IsFlagSet(dwFlags, WC_KEEP_CURSOR_VISIBLE), psScrollY);
+                Status = AdjustCursorPosition(screenInfo, CursorPosition, WI_IsFlagSet(dwFlags, WC_KEEP_CURSOR_VISIBLE), psScrollY);
 
-                    CursorPosition = cursor.GetPosition();
-                }
-            }
-        }
-
-        if (screenInfo.InVTMode())

```

Comment:
With the elimination of all references to `SCREEN_INFORMATION::InVtMode` from this function, I believe that the method can now be made private on `SCREEN_INFORMATION` such that no one else takes a dependency on it going forward.

Reply:
Is this something you'd like me to do now or is it just a note for a followup PR? One of my longer term plans was to see if we could move more of the VT-specific code into the `AdaptDispatch` class, where it could be more easily shared with the Terminal implementation. That would include the current tabstop management, which I think is now the only reason `InVTMode` is required, so we could eventually get rid of that completely.

Reply:
Changing it to private to symbolize that it's no longer accessible I'd like in this PR if possible just to shut the door on other consumers if the longer term plan takes longer than we want.

It should just be moving it down to the private section and then rename/replacing it with _InVTMode or something in the 3-4 locations it is remaining.

I know I'm being a bit confusing in this PR by making some comments, researching them, then resolving them myself. But this one is a legit change request. :)

Reply:
OK. I've now made it private, and moved it down to where the other private boolean getters were declared. Also made a couple of cosmetic changes at the same time. I added an `Is` prefix to the name just to be consistent with the other methods, and slightly restructured the doc comments to match the surrounding code. (550a68587984448210c60bf6298cadb9f0afd94a)

Reply:
Alright, thank you.